From 67f163f569b26917f1aa2586dc0d1e58789aa3f2 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 4 Jan 2021 10:16:52 +0800
Subject: [PATCH] scripts/gcc-wrapper.py: ignore pseudo_files.c:715

fs/incfs/pseudo_files.c:715:10: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I229a33517fbe556abeac4e51b35a8ccd4e91f458
---
 scripts/gcc-wrapper.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/gcc-wrapper.py b/scripts/gcc-wrapper.py
index a4e76c423c15..57bc40f6aa02 100755
--- a/scripts/gcc-wrapper.py
+++ b/scripts/gcc-wrapper.py
@@ -38,6 +38,7 @@ import sys
 import subprocess
 
 allowed_warnings = set([
+    "pseudo_files.c:715", # fs/incfs/pseudo_files.c:715:10: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]
     "km_apphint.c:230", # drivers/staging/imgtec/rogue/km_apphint.c:230:48: warning: division 'sizeof (void *) / sizeof (void)' does not compute the number of array elements [-Wsizeof-pointer-div]
     "file.c:3010", # fs/f2fs/file.c:3010:12: warning: 'f2fs_ioctl_check_project' defined but not used
     "configfs.c:1488", # drivers/usb/gadget/configfs.c:1488:12: warning: 'configfs_composite_setup' defined but not used
-- 
2.35.3

